### PR TITLE
chore: dont fail server db migration on unknown module

### PR DESCRIPTION
**Pull request by m1sterc001guy**

### CODE REVIEW

File: fedimint-server/src/consensus/server.rs [hunk 1]
```diff
@@ -116,7 +117,8 @@ impl ConsensusServer {
         for (module_id, module_cfg) in &cfg.consensus.modules {
             let kind = module_cfg.kind.clone();
             let Some(init) = module_inits.get(&kind) else {
-                bail!("Detected configuration for unsupported module id: {module_id}, kind: {kind}")
```
Comment (dpc) [inline fedimint-server/src/consensus/server.rs:119]: Why is this change needed? We can't be running fedimint server if we don't support any of the modules used for consensus.

Comment (elsirion) [reply fedimint-server/src/consensus/server.rs:119]: Afaik this will just crash at a later point, so not very useful.

Comment (m1sterc001guy) [reply fedimint-server/src/consensus/server.rs:119]: ah I missed that the modules were from consensus. Closing
